### ISSUE TITLE
Reproduce the Crashing app after Tauri upgrade

### ISSUE THREAD

**Issue by tomaszantas** [Altalogy/tari#325]
```
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: "Connection error: transport error caused by:\ntransport error caused by:\nerror trying to connect: tcp connect error: Connection refused (os error 61) caused by:\ntcp connect error: Connection refused (os error 61) caused by:\nConnection refused (os error 61)"', applications/launchpad/backend/src/api/wallet_api.rs:70:84
```

![image](https://user-images.githubusercontent.com/9142942/177124448-8be9500d-08c1-4845-a4a9-f5434fc21488.png)
do not 'unwrap' - handle the error gracefully and return it to the frontend, so it know it should wait

**Comment by tarnas14** [comment]
waiting for tauri upgrade #324 
the crash might be related to production build only

**Comment by tomaszantas** [comment]
Solved with #351
